ingest-v2/src/main/resources/logback.xml

```diff
@@ -0,0 +1,12 @@
+<configuration>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
```

Comment:
Similar question here, when we use it from another connector, need to see if we need to use logback config here, if not we can remove it